index.js

```diff
@@ -12,7 +12,7 @@ class Page {
     * @param {Object.<string, string | function>} reactions The reactions that'll be added to this page.
     * @param {number} index The index of this page in the Menu
      */
-  constructor (name, content, reactions, index) {

```

Comment:
Why removing the spaces before the parenthesis?

Reply:
oh that was my messed up prettier config back from when I didn't know how to configure it correctly, my bad.

Reply:
Oh okay! That's fine I was just curious :) sad this repo is unmaintained though

Reply:
Yeah, I had to use it for a big project so these changes were necessary to make it function better and have less errors.

Too bad it'll probably be deprecated when discord.js 13 releases

Reply:
> sad this repo is unmaintained though
> > Too bad it'll probably be deprecated when discord.js 13 releases

yeah I've not been doing much with this repo since Discord's adding new interaction features like buttons, menus, etc. Not really sure where to take the repo since there's far better ways of handling menus now. Have been intending to refactor/rewrite it for a while. Will probably do something when v13 releases, but no promises. Sorry